pkg/chunkenc/unordered.go

```diff
@@ -89,13 +91,13 @@ func (hb *unorderedHeadBlock) UncompressedSize() int {
 }
 
 func (hb *unorderedHeadBlock) Reset() {
-	x := newUnorderedHeadBlock(hb.format)
+	x := newUnorderedHeadBlock(hb.format, hb.symbolizer)
```

Comment:
should we also reset symbolizer?

Reply:
i might be wrong, I believe we need to reuse symbolizer until we cut the chunk. am I right?

Reply:
yes, that is correct.